app/controllers/polls_controller.rb

```diff
@@ -87,6 +89,13 @@ def remember_id(id)
     cookies.signed[:voted_polls] = JSON.generate(arr.uniq)
   end
 
+  def forgot_id(id)
+    cookies.signed.permanent[:voted_polls] ||= '[]'
```

Comment:
Nope, you will can't vote again by same user becouse of server-side part of check `@already_voted = remembered_ids.include?(id.to_i) || @poll.voters.include?(current_user)` - your id still be in `@poll.voters`. But, funny fing, you can't vote again even when you should, when owner edit poll and mark it ready again cookie on client side still have it as already voted. Will be more correct to not give owner rights to edit poll which have votes, but I do it for educational purpose, so more trouble is good for me.